res/scripts/script-index.js

```diff
@@ -1,24 +1,55 @@
+
+//Creating a new array to store the face images in the deck of card
+var faceCardsArray = new Array(); 
+
+//Creating Two random values to use while switching cards
+var randomOne = Math.floor((Math.random() * 52) + 0);
+var randomTwo = Math.floor((Math.random() * 52) + 0);
+
 //AddEventListener to DOMContentLoaded, so the image waits to load
 //after the page is loaded
+
 document.addEventListener("DOMContentLoaded", function(event) {
 	onload_image();
-  
+
+  //The array has been created so its available in the memory during the game
+  for (var faceCard=0; faceCard<=51; faceCard++){
+    faceCardsArray[faceCard] = new Image(); //Every element of the array is defined as a new image
+    faceCardsArray[faceCard].src = "./res/images/face/" + faceCard + ".png"; //every element source has been set
+
+  }
+ 
 });
 
-var cardOne = document.getElementById("cardOne");
-cardOne.onclick = function() {
-  changeImage(cardOne);
+
+//Capturing which card has been Clicked
+//And it's id has been passed on 
+var cardOne = document.getElementById("displayingImage1");
+cardOne.onclick = function(alt) {
```

Comment:
The Paramter here would be more a event rather then alt.

Reply:
That was experimenting to how to get to calculate which cards been shown